.ci/build_deps.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+set -ex
+
+
+download_and_build_netcdf() {
```

Comment:
@ZedThree it turns out that building Linux with conda packages is more challenging than on Windows! Thanks for you approach, it worked just great here.